velox/exec/Operator.h

```diff
@@ -197,9 +216,11 @@ class OperatorCtx {
   std::optional<Spiller::Config> makeSpillConfig(Spiller::Type type) const;
 
  private:
-  DriverCtx* const FOLLY_NONNULL driverCtx_;
+  DriverCtx* const driverCtx_;
```

Comment:
folly annotation not required?

Reply:
We have discrepancy between tests and code. Test does not have driverCtx.
I plan to follow up this with the change that requires driverCtx all the time and remove the special case. Then we can bring back the annotation.